### PR TITLE
Updates readme

### PR DESCRIPTION
**Pull request by heyitsbryanm**
# Description

Updates the README.md file to mirror our paper doc:
https://paper.dropbox.com/doc/imgix-App-Magento-User-Guide--BdXg19xbOy0FcyYHLZK9l1sKAg-rKmunr1j0zA1fNs0cuVS5

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -37,6 +42,98 @@ php bin/magento cache:clean
...
+
+From the **Magento Page Builder**, users can create a new row/column component (located under Layout section) by dragging it onto the page in the desired location. This is where an imgix image can be added. After, users can click and drag the imgix Image component to the desired row or column.
+
+![Editing with Page Builder](https://assets.imgix.net/sdk/magento/12-editing_page_builder.png?pad=40&w=1520&mask-bg=E8F0F4&mask=corners&corner-radius=12&bg=E8F0F4&auto=compress,format)
+
+From this point, users may follow the same steps as in the Product section to browse, search, and insert an image into this component. Note: in the context of the Page Builder, only one image can be added per component.
+
+![Browsing images in Page Builder](https://assets.imgix.net/sdk/magento/13-browsing_images.png?pad=40&w=1520&mask-bg=E8F0F4&mask=corners&corner-radius=12&bg=E8F0F4&auto=compress,format)
+
+![Selecting an image in Page Builder](https://assets.imgix.net/sdk/magento/14-selecting_an_image_in_page_builder.png?pad=40&w=1520&mask-bg=E8F0F4&mask=corners&corner-radius=12&bg=E8F0F4&auto=compress,format)
```
Comment (sherwinski) [inline README.md:121]: these images are 404ing

File: README.md [hunk 2]
```diff
@@ -37,6 +42,98 @@ php bin/magento cache:clean
...
+- [Height](https://docs.imgix.com/apis/rendering/size/h): Sets the height of the image
+- [Format](https://docs.imgix.com/apis/rendering/format/fm): Converts the image to the specified format
+- [Auto](https://docs.imgix.com/apis/rendering/auto/auto): Performs some baseline optimizations to the image
+- [Crop](https://docs.imgix.com/apis/rendering/size/crop): Controls how the image should be cropped
+- 
+Once any number of these configurations are entered, users can accept these changes by pressing the **Save** button on the top left.
+
+![Saving your rendering API configuration](https://assets.imgix.net/sdk/magento/15-saving_configuration.png?pad=40&w=1520&mask-bg=E8F0F4&mask=corners&corner-radius=12&bg=E8F0F4&auto=compress,format)
+
+![Example output of a saved image in Page Builder](https://assets.imgix.net/sdk/magento/16-page_builder_saved_example.png?pad=40&w=1520&mask-bg=E8F0F4&mask=corners&corner-radius=12&bg=E8F0F4&auto=compress,format)
```
Comment (sherwinski) [inline README.md:139]: same with these

File: README.md [hunk 3]
```diff
@@ -37,6 +42,98 @@ php bin/magento cache:clean
...
+## Customizing an Image
+
+Users may modify the selected image within the component by selecting the Edit button (represented by the gear symbol) when hovering over the image. Under the “imgix image configurations” section, users may elect to customize any of the five parameters shown:
+
+- [Width](https://docs.imgix.com/apis/rendering/size/w): Sets the width of the image
+- [Height](https://docs.imgix.com/apis/rendering/size/h): Sets the height of the image
+- [Format](https://docs.imgix.com/apis/rendering/format/fm): Converts the image to the specified format
+- [Auto](https://docs.imgix.com/apis/rendering/auto/auto): Performs some baseline optimizations to the image
+- [Crop](https://docs.imgix.com/apis/rendering/size/crop): Controls how the image should be cropped
+-
```
Comment (sherwinski) [inline README.md:134]: We can remove this bullet
```suggestion

```

**Review by sherwinski — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (heyitsbryanm) [reply README.md:121]: whoops, forgot to upload them.

they're online now 👍

**Comment by heyitsbryanm** [comment]
@sherwinski it's good for another review

**Review by sherwinski — APPROVED** [review]
Looks good!

**Pull request merged**